hack/version2.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+declare -r UNKNOWN_BRANCH_PREFIX="0.0"
+declare -r MASTER_BRANCH_PREFIX="1.2"
+
+function prefix_from_branch() {
+	declare -r branch_name="$1"
+
+	if [[ "$branch_name" =~ ^([^/]+/)?mce-([[:digit:]]+.[[:digit:]]+) ]]; then
```

Comment:
I've [asked](https://redhat-internal.slack.com/archives/C012L9Y9C79/p1739828571006479) and ACM/MCE don't seem to be using bundle-gen anymore. Given that, and the fact that we don't build OperatorHub bundles for MCE branches anymore, we should be able to eliminate this condition.